<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_ScriptHelp_EffectScope</class>
 <widget class="QWidget" name="P_ScriptHelp_EffectScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">
QToolButton{
	font: 75 10pt &quot;黑体&quot;;
	font-weight:bold;
	padding:2px;
	padding-left:6px;
	border: 1px solid transparent;
	background: rgba(255,255,255,0);
}
QToolButton:hover{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #fdecad, stop:0.3 #fde48c, stop:0.9 #fce58c, stop:1 #fdfadf )
}
QToolButton:pressed{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}
QToolButton:checked{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}
QToolButton:disabled{
	color:#bbbbbb;
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>插件未标注作用域。</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="1">
          <widget class="QToolButton" name="toolButton_battle">
           <property name="text">
            <string>战斗界面</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/DrillPluginInformationReader/Resources/icons/view/Icon_Selected.svg</normaloff>:/DrillPluginInformationReader/Resources/icons/view/Icon_Selected.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="toolButton_menu">
           <property name="text">
            <string>菜单界面</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/DrillPluginInformationReader/Resources/icons/view/Icon_Selected.svg</normaloff>:/DrillPluginInformationReader/Resources/icons/view/Icon_Selected.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButton_map">
           <property name="text">
            <string>地图界面</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/DrillPluginInformationReader/Resources/icons/view/Icon_Selected.svg</normaloff>:/DrillPluginInformationReader/Resources/icons/view/Icon_Selected.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_context">
         <property name="text">
          <string>没有作用域描述。</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../drillplugininformationreader.qrc"/>
 </resources>
 <connections/>
</ui>
